webpack.common.js

```diff
@@ -77,163 +78,176 @@ console.log('npm run watch  --themes_dir=themes/mytheme/client --css_file=myfile
 console.log('npm run build  --themes_dir=themes/mytheme/client --fonts_dir=fontsies')
 console.log('--------------------------------')
 
+/*
+ * Load custom webpack config by command line params
+ */
 let customConfig = {}
 if (WEBPACK_CUSTOM_ADD_PATH_CONFIG) {
   customConfig = require(WEBPACK_CUSTOM_ADD_PATH)
 }
 
-module.exports = merge(
-  {
-    entry: {
-      app: [
-        JS_FILE,
-        CSS_FILE
-      ]
-    },
-    output: {
-      filename: '[name].js',
-      path: path.resolve(
-        DIST_DIR
-      )
-      // crossOriginLoading: 'anonymous'
-    },
-    module: {
-      rules: [
-        {
-          // javascripts
-          test: /\.jsx?$/,
-          // exclude: /node_modules/,
-          use: {
-            loader: 'babel-loader',
-            options: {
-              presets: [
-                '@babel/preset-env',
-                '@babel/react',
-                {
-                  plugins: [
-                    '@babel/plugin-proposal-class-properties'
-                  ]
-                }
-              ],
+const myConfig = merge({
```

Comment:
And take a look to formatting here. As you can see formatting above is
{ merge } = require(...);

while you want to make it here:
merge(
{
}
);

That's 2 different styles of the formatting, I have checked standard --fix and it doesn't style the brackets. Which style of formatting do you prefer?

// [collapse|collapse-preserve-inline|expand|end-expand|none] Put braces on the same line as control statements (default), or put braces on own line (Allman / ANSI style), or put end braces on own line, or attempt to keep them where they are